packages/table/src/table-column/index.ts

```diff
@@ -202,24 +199,22 @@ export default defineComponent({
 
     // eslint-disable-next-line
     instance.columnConfig = columnConfig
-    return {
-      row,
-      r,
-      $index,
-      columnId,
-      columnConfig,
-    }
+    return
   },
   render() {
-    return h(
-      'div',
-      this.$slots.default?.({
-        store: {},
-        _self: {},
-        column: {},
-        row: {},
-        $index: undefined,
-      }),
-    )
+    try {
+      return h(
+        'div',
+        this.$slots.default?.({
+          store: {},
+          _self: {},
```

Comment:
Without passing any data, how could users get data from?

Reply:
Rendering within `el-table-column` helps to generate column structures, cell rendering is called actively in `table-body` by `slots.default`
https://github.com/element-plus/element-plus/blob/fe42f4404e522f8bd6730fb51e0245d014a03512/packages/table/src/table-body/render-helper.ts#L81-L87
https://github.com/element-plus/element-plus/blob/fe42f4404e522f8bd6730fb51e0245d014a03512/packages/table/src/table-body/render-helper.ts#L104-L116
https://github.com/element-plus/element-plus/blob/fe42f4404e522f8bd6730fb51e0245d014a03512/packages/table/src/table-column/render-helper.ts#L111-L131